package.json

```diff
@@ -100,6 +100,7 @@
     "flow-typed": "^2.5.1",
     "husky": "^2.0.0",
     "jest": "^24.7.1",
+    "jest-junit": "^6.4.0",
```

Comment:
Why is this part of this PR?

Reply:
To integrate azure test output with github, you can see what tests failed or not

Reply:
Thanks 👍🏼